### DISCUSSION TITLE
Support for Dark Mode

### DISCUSSION THREAD

**Discussion by kostoman69** [vkurko/calendar#110] [Q&A]
Is there any easy way to changhe theme for Dark mode?
I use the component from Svelte with tailwindcss and all other components conform to light/Dark mode
using the colo scale
...
extend: {
      colors: {
        primary: { "50": "#eff6ff", "100": "#dbeafe", "150": "#def7ec", "200": "#bfdbfe", "300": "#93c5fd", "400": "#60a5fa", "500": "#3b82f6", "600": "#2563eb", "700": "#1d4ed8", "800": "#1e40af", "900": "#1e3a8a", "950": "#def7ec", }
      }
},

Thanks in advance,
Stavros

**Comment by vkurko** [comment]
Although in the Event Calendar the user has the opportunity not only to override the original CSS classes, but also to replace them with their own using the [theme](https://github.com/vkurko/calendar#theme) setting, no one provided their solution in the corresponding [issue](https://github.com/vkurko/calendar/issues/68) that was opened earlier :smile:.

I will try to provide a solution when I have free time.

**Comment by Uhasith** [comment]
Is there any workaround for this?  @vkurko @kostoman69

**Reply by vkurko** [reply]
Dark theme is available since v2.4.0. Please see the [docs](https://github.com/vkurko/calendar#theming).
